docs/OMS-Agent-for-Linux.md

```diff
@@ -547,16 +546,32 @@ The OMS Agent for Linux also listens on port 26000 for CollectD metrics and then
 </filter>
 ```
 
-#### Setup
-
-##### Installation Option
-During installation the option `--collectd` can be used to automatically place the CollectD configuration for versions 5.5+ in `/etc/collectd/conf.d/`, and enable the OMS Agent for Linux filter.
-
-##### Post-Install Option
-If OMS Agent for Linux is already installed you can run the following command: `/opt/microsoft/omsagent/bin/omsadmin.sh -c` to automatically place the CollectD configuration in `/etc/collectd/conf.d/`, and enable the OMS Agent for Linux filter.
-
-
-**Note:** If CollectD is not installed in a default path or is version < 5.5 then these options should not be used, and the files should be placed manually in CollectD configuration directory. **Additionally, CollectD must be restarted after installation**
+#### Steps for setup
+1. To route CollectD data to the OMS Agent for Linux, `oms.conf` needs to be added to CollectD's configuration directory. 
+The destination of this file depends on the Linux flavor of your machine.
+
+    If your CollectD config directory is located in `/etc/collectd.d/`:
+    ```
+    sudo cp /etc/opt/microsoft/omsagent/sysconf/omsagent.d/oms.conf /etc/collectd.d/oms.conf
+    ```
+    If your CollectD config directory is located in `/etc/collectd/collectd.conf.d/`:
+    ```
+    sudo cp /etc/opt/microsoft/omsagent/sysconf/omsagent.d/oms.conf /etc/collectd/collectd.conf.d/oms.conf
+    ```
+    **Note:** For CollectD versions before 5.5 you will have to modify the tags in `oms.conf` as shown above.
+
+2. Copy `collectd.conf` to the desired workspace's omsagent configuration directory.
+    ```
+    sudo cp /etc/opt/microsoft/omsagent/sysconf/omsagent.d/collectd.conf /etc/opt/microsoft/omsagent/<workspace id>/conf/omsagent.d/
```

Comment:
Include the command "sudo chown omsagent:omiusers /etc/opt/microsoft/omsagent/<workspace id>/conf/omsagent.d/collectd.conf"